sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/DefaultOfferRequirementProvider.java

```diff
@@ -470,7 +470,7 @@ private static boolean isPortResource(Protos.Resource resource) {
         // command and user:
 
         Protos.CommandInfo.Builder commandInfoBuilder = executorInfoBuilder.getCommandBuilder()
-                .setValue("export LD_LIBRARY_PATH=$MESOS_SANDBOX/libmesos-bundle/lib && " +
+                .setValue("export LD_LIBRARY_PATH=$MESOS_SANDBOX/libmesos-bundle/lib:$LD_LIBRARY_PATH && " +
```

Comment:
@mrbrowning: It's definitely a thought worth discussion.  This changes something pretty fundamental to the operation of a Framework.  We try to keep each version of a package very well defined.  As long as a change here were statically built into a package I'd be happy, but I worry about N customers defining N different ways to start the executor.  Could be a support problem